doc/schemas/node_openapi.json

```diff
@@ -1370,6 +1370,28 @@
         }
       }
     },
+    "/node/process": {
+      "get": {
+        "responses": {
+          "200": {
+            "content": {
+              "application/json": {
+                "schema": {
+                  "$ref": "#/components/schemas/json"
```

Comment:
nit: if a typed response was returned it would be clearer from the API spec what the format of the JSON is

Reply:
Will fix.